### DISCUSSION TITLE
Code usage permission

### DISCUSSION THREAD

**Discussion by mozartelio** [MarkoStanojevic12/ComponentLibrary#3] [Q&A] [answered]
Dear Mr. Stanojević @MarkoStanojevic12  ,

while I am working on my master thesis as a university student, I need an implementation of a QML live reloading mechanism for creating QML mobile application. I have tried plenty of solutions from the internet but none of them worked in my case and then I found you tutorial on YouTube [https://www.youtube.com/watch?v=Qd9cXY6UH0s](https://www.youtube.com/watch?v=Qd9cXY6UH0s). In the description there is a link to this GitHub repository, so I tried to launch my application using your source code. It gave positive results and now I can use the live reloading. 
Thank you very much for your tutorial and efforts you put into creating it!

Due to the fact that your live reloading source code is available in this public repository, I assume that you let others to use it, but since there is no explicit license in that repository, I need to ask you, if I am allowed to use your live reloading code in my application for master thesis? 

Thank you in advance for your reply.

**Comment by MarkoStanojevic12** [comment, marked as answer]
Hello Anton,
I'm glad you chose QML for your master thesis.
Of course you can use it, that's what it is for! :)  Everything that you see public on my github is for free use.
Good luck on your degree, and happy coding!
